### PR TITLE
#167733633 Integrate HoundCI with badge attached to readme

### PR DESCRIPTION
**Pull request by dchima**
#### What does this PR do?
integrates houndCI with the repository for ESLint review, adds a HoundCI badge to the README

#### Description of Task to be completed?
- Integrated HoundCI to repo
- Added badge to README

#### Any background context you want to provide?
None

#### What are the relevant pivotal tracker stories?
[#167733633](https://www.pivotaltracker.com/story/show/167733633)

#### Screenshots (if appropriate)

screen1

### CODE REVIEW

**Review by Baystef — APPROVED** [review]
The .hound.yml was properly configured and the Hound badge displays correctly on the Readme

File: .hound.yml [hunk 1]
```diff
@@ -0,0 +1,3 @@
+eslint:
+  enabled: true
+  config_file: .eslintrc
```
Comment (King-Success) [inline .hound.yml]: This file needs a line break at the bottom

File: README.md [hunk 2]
```diff
@@ -1,3 +1,5 @@
+[![Reviewed by Hound](https://img.shields.io/badge/ESLint%20Reviewed%20by%20-HoundCI-d16ef5)](https://houndci.com)
```
Comment (King-Success) [inline README.md:1]: Good job with the badge!!

**Comment by King-Success** [comment]
Good job overall, the red arrow on the .hound.yml file signifies that a line break is needed, add that and its a go for me.

**Comment by dchima** [comment]
@King-Success thanks for noting that. Sorted out the eof.

**Review by King-Success — APPROVED** [review]

**Pull request merged**